tests/acl/test_acl_l3v4v6.py

```diff
@@ -0,0 +1,1224 @@
+                if duthost.is_backend_portchannel(k, mg_facts):
+                    backend_pc.append(k)
+            for pc in backend_pc:
+                port_channels[namespace].pop(pc)
+
+            upstream_rifs = upstream_ports_per_dut[namespace]
+            downstream_rifs = downstream_ports_per_dut[namespace]
+            for k, v in list(port_channels[namespace].items()):
+                acl_table_ports[namespace].append(k)
+                acl_table_ports[''].append(k)
+                upstream_rifs = list(set(upstream_rifs) - set(v['members']))
+                downstream_rifs = list(set(downstream_rifs) - set(v['members']))
+            if len(upstream_rifs):
+                for port in upstream_rifs:
+                    # This code is commented due to a bug which restricts rif interfaces to
+                    # be added to global acl table - https://github.com/sonic-net/sonic-utilities/issues/2185
+                    if namespace == DEFAULT_NAMESPACE:
+                        acl_table_ports[''].append(port)
+                    else:
+                        acl_table_ports[namespace].append(port)
+            else:
+                for port in downstream_rifs:
+                    # This code is commented due to a bug which restricts rif interfaces to
+                    # be added to global acl table - https://github.com/sonic-net/sonic-utilities/issues/2185
+                    if namespace == DEFAULT_NAMESPACE:
+                        acl_table_ports[''].append(port)
+                    else:
+                        acl_table_ports[namespace].append(port)
+
+        acl_table_ports_per_dut[duthost] = acl_table_ports
+        downstream_ports[duthost] = downstream_ports_per_dut
+        upstream_ports[duthost] = upstream_ports_per_dut
+
+    t2_information = {
+        "upstream_port_ids": upstream_port_ids,
+        "downstream_port_ids": downstream_port_ids,
+        "downstream_port_id_to_router_mac_map": downstream_port_id_to_router_mac_map,
+        "upstream_port_id_to_router_mac_map": upstream_port_id_to_router_mac_map,
+        "acl_table_ports": acl_table_ports_per_dut
+    }
+
+    return t2_information
+
+
+@pytest.fixture(scope="module")
+def setup(duthosts, ptfhost, rand_selected_dut, rand_unselected_dut, tbinfo, ptfadapter, topo_scenario):
+    """Gather all required test information from DUT and tbinfo.
+
+    Args:
+        duthosts: All DUTs belong to the testbed.
+        rand_one_dut_hostname: hostname of a random chosen dut to run test.
+        tbinfo: A fixture to gather information about the testbed.
+
+    Yields:
+        A Dictionary with required test information.
+
+    """
+
+    mg_facts = rand_selected_dut.get_extended_minigraph_facts(tbinfo)
+    topo = tbinfo["topo"]["type"]
+
+    vlan_ports = []
+    vlan_mac = None
+    # Need to refresh below constants for two scenarios of M0
+    global DOWNSTREAM_DST_IP, DOWNSTREAM_IP_TO_ALLOW, DOWNSTREAM_IP_TO_BLOCK
+
+    if topo == "mx":
```

Comment:
If the test is limited to `t0` and `t1`, do we still need to check `mx` and `m0` ?